terms/semisupervised_mm.tex

```diff
@@ -0,0 +1,99 @@
+
+In reality however it is not that simple. Blindly opting for Semi-Supervised methods\index{semi-supervised method} for any specific task can often lead to worse results than a Supervised solution\index{supervised solution} \citep{zhu2009introduction}. 
+
+This is due to the assumption we make when handling the unlabelled data\index{unlabelled data} in our dataset\index{dataset}. Since our Semi-Supervised model\index{semi-supervised model} relies heavily on this assumption, a wrong one can lead to a significant decrease in performance and accuracy. Careful evaluation of the data is a must before committing to which type of learning algorithms\index{algorithm} to use.
+
+\section{Semi-Supervised Learning in Algorithms \& Applications}\label{sec:applications}
+
+\newthought{Semi-Supervised Learning}\index{semi-supervised learning} can be found in various practical applications, including i) image searching\index{image searching}, ii) genomics\index{genomics}, iii)natural language processing\index{natural language processing} and iv) speech analysis\index{speech analysis}. It can be integrated inside of well-known algorithms\index{algorithm}, each of which having its own advantages and disadvantages, and must be used dependently on the application in question.
+
+A simple yet effective algorithm making use of Semi-Supervised Learning\index{semi-supervised learning} is known as the Self-Training Model\index{self-training model} \citep{mcclosky2006effective, zhu2009introduction}. It is a wrapper method\index{wrapper method}, capable of wrapping itself around other algorithms\index{algorithm} without altering their inner workings. It also comes with a crucial limitation; small errors occurring in the initial training iterations\index{training iteration} can get reinforced throughout the rest of the training.
+
+To combat this limitation, among other improvements, one also finds other, more complex algorithms\index{algorithm}. Transductive SVMs\index{transductive SVM} \citep{bennett1999semi} are Support Vector Machines\index{support vector machine} embedded with Semi-Supervised Learning\index{semi-supervised learning}.These methods however have difficulty scaling to large amounts of data. Graph-Based Methods\index{graph-based method} \citep{goldberg2006seeing, zhu2009introduction} are some of the most used techniques\index{semi-supervised technique}. Labelled information\index{labelled information} is spread through the graph from labelled\index{labelled node} to unlabelled nodes\index{unlabelled node}, connecting similar observations. One also finds Neural Network\index{neural network} solutions such as Generative Models\index{generative model} and Deep Generative Models\index{deep generative model} \citep{zhu2009introduction, kingma2014semi}, which are capable of allowing a more robust set of features to be used than linear embedding\index{linear embedding} used by other solutions.
+
+\index{class options|)}
```

Comment:
wrong this is a multipage index ... look at https://en.wikibooks.org/wiki/LaTeX/Indexing#Multiple_pages on how to do this.  Also this should be named "Semi-Supervised Learning" and should have a matching start tag,